### DISCUSSION TITLE
board support

### DISCUSSION THREAD

**Discussion by omaribrahim073** [FPGAwars/icestudio#818] [New Boards Requests]
i am new in fpga world and i got Altera max II EPM240 cpld can you add it it support

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.

**Reply by omaribrahim073** [reply]
This aged well
I'm happy that your still working on this masterpiece.
hope you add the board.
The reason i want you to add this board family is because any beginner will usually buy small board like it so more people will use your project.
